### DISCUSSION TITLE
Secret and ConfigMap hardcoded names

### DISCUSSION THREAD

**Discussion by marioosh11** [argoproj/argo-helm#1800] [Q&A] [answered]
Hi Argo Lions
Is it possible to avoid hard-coded names for objects below?  We would like to have possibility to change names and use different ones in case when we use it in different helm charts.
```
kind: Secret
metadata:
  name: argo-rollouts-notification-secret
```

and
```
kind: ConfigMap
metadata:
  name: argo-rollouts-notification-configmap
```

BR
Mario

**Comment by pdrastil** [comment, marked as answer]
Hi, it's not possible as most of Argo CD components hardcode these inside the controllers. You can find the ones you are asking about here: https://github.com/argoproj/argo-rollouts/blob/e64637171f6555a50c3bdc51820adc3212a2e9b5/utils/record/record.go#L47

**Reply by marioosh11** [reply]
So they should be handled by them directly in code am I right?

**Reply by pdrastil** [reply]
Yes, code is watching exactly for these names.
